src/components/header/index.js

```diff
@@ -0,0 +1,31 @@
+import { NavigationMenu } from "../navigation-menu";
+import { isMobile } from "../../utils";
+
+const Header = () => {
+  return isMobile() ? (
+    <div className="fixed top-0 left-0 z-40 p-10 shrink-0 flex items-center justify-between w-full">
+      <NavigationMenu />
+      <div className="flex">
+        {/* Placeholder for Language Picker */}
+        <div className="text-4xl">En</div>
```

Comment:
I think we can remove this for now. The first launch of this website will likely be without translations.